app/models/validators/asylum_support_decision_validator.rb

```diff
@@ -14,6 +15,16 @@ class AsylumSupportDecisionValidator < SimpleDelegator
   validates :tribunal_decision_judges, presence: true
   validates :tribunal_decision_landmark, presence: true
   validates :tribunal_decision_reference_number, presence: true
-  validates :tribunal_decision_sub_category, presence: true
+  validates :tribunal_decision_sub_category, tribunal_decision_sub_category: true
```

Comment:
Further to the recent conversation about this line, my point was that `validates :tribunal_decision_sub_category, tribunal_decision_sub_category: true` does not say how the `tribunal_decision_sub_category` validated. Might a better name for this validator be `tribunal_decision_sub_category_relates_to_parent`?